### DISCUSSION TITLE
Problem with Data from ATEM

### DISCUSSION THREAD

**Discussion by eonlink** [josephdadams/TallyArbiter#423] [Q&A]
Hello,

I'm having trouble with the data being received from a BM ATEM mini extreme.

Running TA 3.0.3
ATEM mini extreme Software version 8.6.1

I can connect to the ATEM but Tally Data returns:

Source: ATEM Mini Extreme Address: undefined Label: undefined PVW: False PGM: False

This happen on any cut over to another camera.

Could someone point me in a direction to get this resolved?  
I can get the ATEM working correctly on 2.16 but not 3.0 and above.
Nothing in the logs help diagnose the problem.

config.json snippet
  {
   "name": "ATEM Mini Extreme",
   "data": {
    "addresses": [
     "192.168.2.120",
     "ATEM Mini Extreme._blackmagic._tcp.local"
    ],
    "ip": "192.168.2.120",
    "me_onair": [
     "1",
     "2",
     "3",
     "4"
    ],
    "cut_bus_mode": "off"
   },
   "sourceTypeId": "44b8bc4f",
   "reconnect": true,
   "enabled": true,
   "id": "ac7b391e",
   "max_reconnects": 5,
   "connected": false
  }

**Comment by hrueger** [comment]
Hi, this is the same problem as in #422.

Do you have any listener clients connected? You could also try the Web Tally Client. I'm pretty sure that those work, the only problem is that the Tally Data is not printed in the logs correctly.

**Comment by MatteoGheza** [comment]
Tally Data currently needs to be fixed, it returns "`Address: undefined Label: undefined PVW: False PGM: False`" for every Tally update.
We'll fix it in a few days.
Could you try checking tally updates in the devices page?

**Reply by eonlink** [reply]
Thanks for the update.
I got it working and checking of the tally updates in the devices page

**Comment by Jocke4f** [comment]
The ATEM connection issue was solved in https://github.com/josephdadams/TallyArbiter/releases/tag/v3.0.5.
